### PR TITLE
#94: Remove use of MPI_COMM_WORLD in Krino package

### LINKED ISSUE #94: Krino: Remove the use of `MPI_COMM_WORLD`

### PR DESCRIPTION
**Pull request by thearusable**
Close #94

### CODE REVIEW

File: packages/krino/krino/region/Akri_Startup.cpp [hunk 1]
```diff
@@ -84,7 +84,7 @@ Startup::Startup(int argc, char ** argv)
     throw std::runtime_error("MPI_Init failed");
   }
 
-  stk::EnvData::instance().m_parallelComm = MPI_COMM_WORLD;
```
Comment (JacobDomagala) [inline packages/krino/krino/region/Akri_Startup.cpp:87]: This function calls `MPI_Init` so it makes sense to use `MPI_COMM_WORLD` here (there's no comm split happening)

File: packages/krino/krino/krino_lib/Akri_BoundingBoxMesh.hpp [hunk 2]
```diff
@@ -75,7 +76,7 @@ class BoundingBoxMesh {
 public:
   BoundingBoxMesh(stk::topology element_topology, const std::vector<std::string>& rank_names = std::vector<std::string>());
   void set_domain(const BoundingBoxType & mesh_bbox, const double mesh_size, const int pad_cells = 0);
-  void populate_mesh(stk::ParallelMachine pm = MPI_COMM_WORLD, const stk::mesh::BulkData::AutomaticAuraOption auto_aura_option = stk::mesh::BulkData::AUTO_AURA);
```
Comment (JacobDomagala) [inline packages/krino/krino/krino_lib/Akri_BoundingBoxMesh.hpp:78]: I think having MPI_COMM_WORLD as default value is ok, if the user wants, he'll just use different comm

**Comment by JacobDomagala** [comment]
I'm ok with the changes, but I think the use of `MPI_COMM_WORLD` is justified here.

**Comment by thearusable** [comment]
@JacobDomagala Ok, in this case I think it would be good to close this PR. 

@stmcgovern What do you think?

**Comment by stmcgovern** [comment]
Default argument use of `MPI_COMM_WORLD` in Krino is fine.

**Pull request closed without merging**